tests/test_e2e_10_mef_eline.py

```diff
@@ -386,13 +393,23 @@ def test_040_disable_circuit_should_remove_openflow_rules(self):
         evc1 = data['circuit_id']
         time.sleep(10)
 
-        # disable the circuit
+        # It verifies EVC's status
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['enabled'] is True
+
+        # It disables the circuit
         payload = {"enable": False}
-        api_url += evc1
-        response = requests.patch(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        # api_url += evc1
```

Comment:
```suggestion
```

Reply:
Resolved with commit [936b436](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/936b43664e41092d237ea2fa397f8db8321023ee)